docs/APIReference-ContentBlock.md

```diff
@@ -74,7 +74,7 @@ supplied text.
   </li>
   <li>
     <a href="#getcharacterlist">
-      <pre>getCharacterList(): List<CharacterMetadata></pre>
+      <pre>getCharacterList(): List&lt;CharacterMetadata&gt;</pre>
```

Comment:
Unfortunately we use MDX (JSX in Markdown) now so it thinks `<CharacterMetadata>` is a React component and I have to change it to use the character entity references.